src/devices/device_models/common/duration_constraints.jl

```diff
@@ -13,11 +13,14 @@ function device_duration_retrospective(ps_m::CanonicalModel,
     time_steps = model_time_steps(ps_m)
     name_up = Symbol(cons_name,:_up)
     name_down = Symbol(cons_name,:_down)
-
-    ps_m.constraints[name_up] = JuMPConstraintArray(undef, set_names, time_steps)
-    ps_m.constraints[name_down] = JuMPConstraintArray(undef, set_names, time_steps)
-
-
+    var1 = var(ps_m, var_names[1])
```

Comment:
Let's give the variables names that are a bit more representative, in my original example I didn't do it but in a second look at it probably is best to given them some sensical name (not a single letter). Also, if you can add a Docstring with the function constraint represented in common that would be great.